pkg/command/context.go

```diff
@@ -200,13 +200,13 @@ func createCtx(_ *cobra.Command, args []string) (err error) {
 	}
 
 	// Sync all required plugins
-	_ = syncContextPlugins()
+	_ = syncContextPlugins(ctx.Target)
 
 	return nil
 }
 
-func syncContextPlugins() error {
-	err := pluginmanager.SyncPlugins()
+func syncContextPlugins(target configtypes.Target) error {
```

Comment:
What if the target is the new `tae`?  maybe we should just `return` in this case since there are no plugins to sync.

Reply:
Actually, I am wrong.  There could be `tae` context plugins.  So you have things correct.  Sorry.